superset-frontend/plugins/legacy-plugin-chart-partition/src/controlPanel.tsx

```diff
@@ -322,6 +322,10 @@ const config: ControlPanelConfig = {
                 ['2 years', t('2 years')],
                 ['156 weeks', t('156 weeks')],
                 ['3 years', t('3 years')],
+                ['208 weeks ago', t('208 weeks ago')],
```

Comment:
@john-bodley Is there are reason to keep the time period in weeks when they match the year? Why not use weeks only when the number of weeks is not a multiple of 52?

Reply:
As @michael-s-molina mentioned, the custom time shift option can vary in different scenarios. To ensure consistency, it is advisable to override the custom option by utilizing the chartControlRegistry on the vendor side.

Reply:
> john-bodley Is there are reason to keep the time period in weeks when they match the year? Why not use weeks only when the number of weeks is not a multiple of 52?

In order to effectively analyze daily and weekly data and account for seasonal trends, it is essential to incorporate the weekly offset.

FYI:
Certain industries or businesses may experience specific patterns or trends on a weekly basis. For example, in the hospitality industry, weekends may have higher demand compared to weekdays. By using weeks as the time period, analysts can effectively capture these patterns and make more accurate predictions or recommendations.

Reply:
Thanks @justinpark for the additional context. @john-bodley I wonder if we should make these configurable somehow.